mlrun/runtimes/sparkjob.py

```diff
@@ -232,7 +232,12 @@ def _run(self, runobj: RunObject, execution: MLClientCtx):
                             meta.name, driver, status
                         )
                     )
-                    logger.info("use .watch({}) to see logs".format(meta.name))
```

Comment:
Why you removed that ? it's not true ?

Reply:
it's both wrong and no longer relevant as the log is done on the mlrun-api server side